src/forms/FormFieldAdvancedSelection/AdvancedSelection.stories.tsx

```diff
@@ -30,6 +30,7 @@ export const Playground = (): ReactElement => {
 	);
 	const getOptionsLimit = text("Get options limit", "5");
 	const createNewOptionsKnob = boolean("Create new option", true);
+	const selectLimit = number("Select limit", 5);

```

Comment:
We still want devs / users to be able to fully play around with the component in the playground. If we keep this knob as number, they won't be able to test how to select all options (unless they know the max number of options and put that number here). I think we should keep this as string and parse it to number in line 98. I'll add an example there.